File: features/fintech/instant_loan/src/main/java/com/tokopedia/instantloan/router/InstantLoanRouter.java
package com.tokopedia.instantloan.router;

import android.content.Context;
import android.content.Intent;

/**
 * Created by lavekush on 20/03/18.
 */

public interface InstantLoanRouter {
    Intent getInstantLoanActivityIntent(Context context);
    Intent getLoginIntent(Context context);
}

File: features/fintech/instant_loan/src/main/java/com/tokopedia/instantloan/router/InstantLoanRouter.kt
package com.tokopedia.instantloan.router

import android.content.Context
import android.content.Intent

/**
 * Created by lavekush on 20/03/18.
 */

interface InstantLoanRouter {
    val isInstantLoanEnabled: Boolean
    fun getLoginIntent(context: Context?): Intent
}
